### DISCUSSION TITLE
自己精简编译后还是太大,有没有大佬给个模块说明,哪些还能精简的

### DISCUSSION THREAD

**Discussion by GitCourser** [AlistGo/alist#6671] [Q&A]
把程序丢路由器里,但空间不够,以前的旧版不到40M的还能用,现在越来越大了,
能否给个精简的思路说明, 比如哪些模块在哪里,要如何删除,例如离线下载,单点登录之类的
下面是我精简编译后的 mipsle 版大小
去掉前端:                        55,896,304
再去掉不用的储存驱动:  50,926,256

ps:不能用upx,太占运行内存

**Comment by pongfcnkl** [comment]
建议还是放过路由器吧
